test/Docker.DotNet.Tests/IImageOperationsTests.cs

```diff
@@ -69,6 +69,19 @@ public async Task CreateImageAsync_TaskCancelled_ThowsTaskCanceledException()
             Assert.True(createImageTask.IsCanceled);
         }
 
+        [Fact]
+        public async Task CreateImageAsync_ErrorResponse_ThrowsDockerApiException()
+        {
+            await Assert.ThrowsAsync<DockerApiException>(() => _dockerClient.Images.CreateImageAsync(
```

Comment:
FYI: You do not need to `await` it. You can just return the task.

Reply:
The redundant async/await are removed.